examples/greeting/Cargo.toml

```diff
@@ -12,4 +12,5 @@ name = "greeting"
 path = "src/main.rs"
 
 [dependencies]
-fluence = { git = "https://github.com/fluencelabs/rust-sdk" }
+#fluence = { git = "https://github.com/fluencelabs/rust-sdk" }
+fluence = { path = "/Users/mike/dev/work/fluence/wasm/rust-sdk/fluence" }
```

Comment:
It might be easier for you to have a `patch` section in workspace Cargo.toml. Not sure if it applies to examples, though